### ISSUE TITLE
Breadcrumb - Design update

### ISSUE THREAD

**Issue by babsdenney** [department-of-veterans-affairs/vets-design-system-documentation#2068]
## Description
In the last staging review, some design issues were found for the breadcrumbs. Below are notes from this [ticket #64284](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64284)

Should: This is a departure from the USWDS, but the last item in the breadcrumbs indicating the current page should be a link with aria-current="page" applied to the a element. That makes it consistent with the other elements in the nav, including being focusable as screen reader users tab through the elements. @rsmithadhoc's https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1458#issuecomment-1671807757 has a good thought process to back up this decision, and it's supported by the [W3C's ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/) and some examples from [Scott O'Hara](https://scottaohara.github.io/a11y_breadcrumbs/) and [Léonie Watson](https://tink.uk/using-the-aria-current-attribute/).

In terms of styling, I would suggest we still maintain material honesty (unlike the W3C and Scott O'Hara examples): the current page element should still look like a link. It might make sense to style the link differently (bold? different color? different underline style?) but it should still be recognizably linky. Hopefully that's an improvement that can also be contributed back to the USWDS.

TLDR; The current page element should still look like a link. How should we make it look like a link but still visually indicate the current page. Brian has some suggestions above. 

## Details
See the description above for links and references.

## Tasks
- [ ] Conduct small audit if necessary (if component already exists and we are building a new version)
- [ ] Review DST backlog for outstanding design issues with this component, if necessary (if this component already exists and we are building a new version)
- [ ] Create designs for component
- [ ] Review designs with PO and/or DSC
- [ ] Review designs with an accessibility specialist
- [ ] Review designs with DST members (Carol can help schedule this)
- [ ] Address any comments from reviews, if necessary
- [ ] Comment on this ticket with any accessibility considerations engineers may need to know
- [ ] Comment on this ticket with content specifications (e.g. labels and error messages)
- [ ] Comment on this ticket with a link to the designs and post in DST Slack channel

## Acceptance Criteria
- [ ] Component design is complete and has been reviewed
- [ ] Accessibility considerations have been added to this ticket, if necessary
- [ ] Content specifications have been added to this ticket, if necessary
- [ ] Link to design has been added to this ticket and shared in Slack

**Comment by babsdenney** [comment]
Development ticket to implement this new breadcrumb design.
[Breadcrumb - Design update #2117](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2117)
